### ISSUE TITLE
Send invoice - Incomplete options on payment button when sending an invoice

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#43145]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.79-3
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Issue reported by:** Applause - Internal Team

**Issue found when executing PR** https://github.com/Expensify/App/pull/41859
## Action Performed:
1. Open the app in mWeb
2. Go to FAB
3. Select "Send invoice"
4. Send Invoice
5. Go to user who needs to pay this invoice

## Expected Result:
Verify that correct settlement button appears in preview and inside of the report header:
* Personal Card or Bank account
* Business Card or Bank account
* Forward to someone else

## Actual Result:
The menu of the button presents only the option "Pay $x as an individual" and the options like the desing are not displayed

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [x] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/Expensify/App/assets/78819774/53fc3d1f-3341-4e76-93a5-3be9c9238390

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by ikevin127** [comment]
**TLDR:** We need to figure out if the offending PR actually missed something functionality-wise, or it's just a design-related misunderstanding.

Posted https://github.com/Expensify/App/pull/41859#issuecomment-2150906716 in offending PR because we need to actually figure out whether the PR's changes are the actual expected result or this issue's **expected result** is what we want, since the PR's changes were approved by the **design team**.

**Comment by AndrewGable** [comment]
Seems like a daily bug, not a deploy blocker. We should fix but don't think we need to block deploy on this. Assigning the relevant members

**Comment by VickyStash** [comment]
As I mentioned in [this comment](https://github.com/Expensify/App/pull/41859#issuecomment-2151541003) it's an expected result to have only `Pay as an individual `option for now. Other options will be implemented in the future. For example, there is a separate [issue](https://github.com/Expensify/App/issues/41974) for the `Pay as business` option implementation.

**Comment by ikevin127** [comment]
Given this information, I think it's safe to say that we can close this out as NAB nor a regression.

@cristipaval Most likely you have even more context on this new UI / functionality and would be able confirm on this.

**Comment by cristipaval** [comment]
yes, this is expected for now. An upcoming PR will add one more option
